Makefile

```diff
@@ -2,7 +2,7 @@ dev_build_version=$(shell git describe --tags --always --dirty)
 # to_check is all code in this repo that we want to run checks on
 # (it is all Go code in here, but intentionally excludes the
 # vendor folder contents)
-dirs_to_check=$(shell find . -maxdepth 1 -mindepth 1 -type d | grep -v .git | grep -v vendor | grep -v .images)
+dirs_to_check=./warehouse ./config
```

Comment:
This is slightly risky: if you add a new folder, it's easy to forget to update this line, and you then won't be getting any of these checks for the new stuff.